README.md

```diff
@@ -0,0 +1,177 @@
+**[Leaderboard]** is a javascript,html, and css project that retrieves leader baord scores from the leaderboard api
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+    <li><a href="https://reactjs.org/">CSS</a></li>
+    <li><a href="https://reactjs.org/">JAVASCRIPT</a></li>
+        <li><a href="https://reactjs.org/">webpack</a></li>
+  </ul>
+</details>
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Leaderboard initial static]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+> Add a link to your deployed project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+<!-- ## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+- Clone this repository using the its url and git using the your terminal.
+- cd into the project using the terminal.
+- RUN npm install to install the nodemofulesal.
+- After installing the node_modules, RUN npm start. -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+To get a local copy up and running, follow these steps.
+- Clone this repository using the its url and git using the your terminal.
+- cd into the project using the terminal.
+- RUN npm install to install the nodemofulesal.
+- After installing the node_modules, RUN npm start to fire up the project
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Luzinda DOuglas**
+
+- GitHub: [@githubhandle](https://github.com/LdouglasOt)
+- Twitter: [@twitterhandle](https://twitter.com/LdouglasOt)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/LdouglasOt)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
```

Comment:
- [ ] It is not a good practice to leave incomplete sentences in your README, Kindly try to make your README more professional by filling the project details in all your README headers, 😉😉😉

   _Remember, It is important to provide complete and clear information in a README to make the project easily understandable for others._

Reply:
In the first pull request for the first issue tackled, I was asked to remove extra details from the readme file, kindly elaborate more on this. 
Thanks, kind regards

Reply:
I was asked to have information limited to this specific project.